source/server-selection/server-selection.rst

```diff
@@ -521,6 +527,30 @@ Eligibility MUST be determined from ``tag_sets`` as follows:
 - If the ``tag_sets`` list is not empty and no tag set in the list matches any
   candidate server, no servers are eligible servers.
 
+hedge
+`````
+
+The read preference ``hedge`` parameter is a document that configures how the
+server will perform hedged reads. It consists of the following keys:
+
+- ``enabled``: Enables or disables hedging
+
+To enable hedging with server-side defaults, an empty ``hedge`` document must be
```

Comment:
After discussing this with @jyemin, I've decided to change this and the following paragraphs in commit b279fe4. When providing an API, allowing the user to build an empty hedge document makes no sense, as the user may have trouble understanding how the server behaves. This also removes API ambiguity where the `enabled` hedge option would effectively become a ternary bool (unset, `true`, and `false`).

If a driver allows the user to specify an arbitrary document for the `hedge` option, an empty document is still permitted.